trin-execution/src/storage/evm_db.rs

```diff
@@ -21,14 +21,12 @@ use super::{
     trie_db::TrieRocksDB,
 };
 
-const REVERSE_HASH_LOOKUP_PREFIX: &[u8] = b"reverse hash lookup";
-
 #[derive(Debug, Clone)]
 pub struct EvmDB {
     /// State config
     pub config: StateConfig,
     /// Storage cache for the accounts used optionally for gossiping.
-    pub storage_cache: HashMap<Address, HashSet<B256>>,
+    pub storage_cache: HashMap<B256, HashSet<B256>>,
```

Comment:
nit: might be worth updating the comment to indicate that key is keccak of address